### PR TITLE
[WFLY-8612] Create test case for Elytron Ldap Realm

### PR DESCRIPTION
**Pull request by olukas**
https://issues.jboss.org/browse/WFLY-8612
https://issues.jboss.org/browse/JBEAP-10465

### CODE REVIEW

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/realm/LdapRealmTestCase.java [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            },
+            certificatePassword = "secret")
+    //@formatter:on
+    static class LDAPServerSetupTask implements ServerSetupTask {
+
+        private DirectoryService directoryService;
+        private LdapServer ldapServer;
+
+        /**
+         * Creates directory services, starts LDAP server and KDCServer
```
Comment (kwart) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/realm/LdapRealmTestCase.java]: C/P - there is no KDC here.

File: testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/realm/LdapRealmTestCase.java [hunk 2]
```diff
@@ -0,0 +1,360 @@
...
+            }
+            final ManagedCreateLdapServer createLdapServer = new ManagedCreateLdapServer(
+                    (CreateLdapServer) AnnotationUtils.getInstance(CreateLdapServer.class));
+            Utils.fixApacheDSTransportAddress(createLdapServer, Utils.getSecondaryTestAddress(managementClient, false));
+            ldapServer = ServerAnnotationProcessor.instantiateLdapServer(createLdapServer, directoryService);
+            ldapServer.start();
+        }
+
+        /**
+         * Stops LDAP server and KDCServer and shuts down the directory service.
```
Comment (kwart) [inline testsuite/integration/elytron/src/test/java/org/wildfly/test/integration/elytron/realm/LdapRealmTestCase.java]: Maybe remove the javadoc in `setUp` and `tearDown` at all :)

**Review by kwart — APPROVED** [review]
Found just a C/P issue in javadoc.

**Comment by olukas** [comment]
@kwart Fixed. Thanks

**Review by kabir — APPROVED** [review]
Approving paperwork

**Pull request merged**